sig-network/api-extensions-best-practices.md

```diff
@@ -0,0 +1,141 @@
+# API Extensions - Best Practices
+
+It is sometimes desireable to build new networking features as [CRDs] rather
+than [in-tree]. Reasons may include (but aren't limited to):
+
+* A desire to be free of lockstep with the [KEP] process
```

Comment:
the KEP process serves a purpose, this get the impression the process or the purpose is wrong, is far from ideal but at least help the project to be sustainable ... I'm not in favor of this sentence

Can we rephrase it as "move at different pace that the core project"? or something like that?

Reply:
Sure thing!

```suggestion
* A need to move at a different pace than the core project
```